app/assets/javascripts/news.js.erb

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function() {
```

Comment:
If you could comment what this does / or give it a more descriptive title. Also a comment as to why you are using jQuery.